src/typechecker/lisp-type.lisp

```diff
@@ -19,15 +19,16 @@
 ;;; Lisp types for coalton types
 ;;;
 
-(defgeneric lisp-type (ty env)
+(defgeneric lisp-type (ty env &key use-function-entries)
```

Comment:
you might make the generic function `%lisp-type` or `lisp-type-gf` and not have the annoying KW arg, then define

```
(defun lisp-type (ty env &key use-fu)
  (%lisp-type ty env use-fn))
```

Not required, but would make the methods nicer without having to repeat yourself with the optional values.